xds/third_party/xds/import.sh

```diff
@@ -26,26 +26,26 @@ TARGET_PROTO_BASE_DIR=src/main/proto
 # Sorted alphabetically.
 FILES=(
 udpa/annotations/migrate.proto
-xds/annotations/v3/migrate.proto
 udpa/annotations/security.proto
-xds/annotations/v3/security.proto
 udpa/annotations/security.proto
-xds/annotations/v3/security.proto
 udpa/annotations/sensitive.proto
-xds/annotations/v3/sensitive.proto
 udpa/annotations/status.proto
-xds/annotations/v3/status.proto
 udpa/annotations/versioning.proto
-xds/annotations/v3/versioning.proto
-xds/data/orca/v3/orca_load_report.proto
-xds/service/orca/v3/orca.proto
 udpa/type/v1/typed_struct.proto
-xds/type/v3/typed_struct.proto
+xds/annotations/v3/migrate.proto
```

Comment:
This comment is for the whole commit "Sort xds/import.sh protos alphabetically". 

- have we decided that the strict alphabetical order is better than for example having related protos together? e.g. the 2 versions of migrate.proto together?
- we will eventually be dropping support for v2 xds/protos? If it is imminent is this reordering worthwhile?

Reply:
1. Yes, there's even a comment on top of the import, same as with all other `import.sh` files. We just didn't do it in xds/import.sh for some reason. https://github.com/grpc/grpc-java/blob/538db03d56630df042c44c2e81363be924a2be8f/xds/third_party/xds/import.sh#L26
2. Don't know if or when we'll drop them. Reordering was cheap for me - select lines -> ctrl+F5 to sort. Didn't consider this to be a price.